pkg/machines/components/vnc.jsx

```diff
@@ -83,16 +105,69 @@ class Vnc extends React.Component {
         console.error('VncConsole failed to init: ', detail, this);
     }
 
+    onExtraKeysDropdownToggle() {
+        this.setState({
+            isActionOpen: !this.state.isActionOpen
+        });
+    }
+
     render() {
-        const { consoleDetail } = this.props;
-        const { path } = this.state;
+        const { consoleDetail, vm, onAddErrorNotification, config } = this.props;
+        const { path, isActionOpen } = this.state;
         if (!consoleDetail || !path) {
             // postpone rendering until consoleDetail is known and channel ready
             return null;
         }
+        const connnectionName = vm.connnectionName;
+        const vmId = vm.id;
 
         const credentials = consoleDetail.password ? { password: consoleDetail.password } : undefined;
         const encrypt = this.getEncrypt();
+        const dropdownItems = [
+            ...['Delete', 'Backspace'].map(key => {
+                return (
+                    <DropdownItem
+                        key={cockpit.format("ctrl-alt-$0", key)}
+                        onClick={() => {
+                            return domainSendKey(connnectionName, vmId, [Enum.KEY_RIGHTCTRL, Enum.KEY_RIGHTALT, Enum[cockpit.format("KEY_$0", key.toUpperCase())]])
+                                    .fail(ex => onAddErrorNotification({
+                                        text: cockpit.format(_("Failed to send key Ctrl+Alt+$0 to VM $1"), key, name),
+                                        detail: ex.message
+                                    }));
+                        }}>
+                        {cockpit.format("Ctrl+Alt+$0", key)}
```

Comment:
I think this should be localized.

Reply:
Also with the last item, which cannot be directly localized because it is used in may, but you will figure something out :)